lib/ethereum/cw_ethereum.dart

```diff
@@ -131,4 +131,9 @@ class CWEthereum extends Ethereum {
   void updateEtherscanUsageState(WalletBase wallet, bool isEnabled) {
     (wallet as EthereumWallet).updateEtherscanUsageState(isEnabled);
   }
+
+  @override
+  Future<String> fetchEnsAddress(WalletBase wallet, String name) async {
+    return (wallet as EthereumWallet).fetchEnsAddress(name);
+  }
```

Comment:
ENS names can contain addresses for different assets other than Ethereum (example: https://app.ens.domains/vikrant.eth). Move this to `/lib/entities/ens_record.dart`